packages/dd-trace/src/appsec/iast/vulnerabilities-formatter/index.js

```diff
@@ -43,6 +43,8 @@ class VulnerabilityFormatter {
     const valueParts = []
     let fromIndex = 0
 
+    if (evidence.value == null) return
```

Comment:
@uurien little eyes por favor (careful it's a double equal)

Reply:
A little background on the double equal usage: I found that earlier in the call chain, there's a check for `evidence.value === undefined`, in which case it aborts, so in reality it can never be `undefined` at this location as the code current looks. However, since the _intention_ is that an `undefined` value should never reach this location, I used `==` in case the call chain ever changes in the future.

Reply:
This method returns `{ valueParts }`, even when the array is empty. I think that the right solution is:
```suggestion
    if (evidence.value == null) return { valueParts }
```